ui_testing/pages/testplan_page.py

```diff
@@ -240,3 +240,13 @@ def get_testunit_quantification_limit(self, testunits, testunit_display_name):
                 quantification_limit = testunit['Quantification Limit']
                 return quantification_limit
 
+    def edit_test_unit_quantification_limits(self, upper, lower):
+        self.navigate_to_testunits_selection_page()
+        elems = self.base_selenium.find_elements('general:col_6')
+        old_upper = self.base_selenium.find_element_in_element(source=elems[4], destination_element='general:input')
+        old_lower = self.base_selenium.find_element_in_element(source=elems[5], destination_element='general:input')
+        old_lower.clear()
+        old_upper.clear()
+        old_upper.send_keys(upper)
+        old_lower.send_keys(lower)
```

Comment:
user `https://github.com/modeso-open-source/1lims-automation/blob/master/ui_testing/pages/base_selenium.py#L310`

Reply:
I removed it and used https://github.com/modeso-open-source/1lims-automation/blob/LIMS5300-4426/ui_testing/pages/testplan_page.py#L232

Reply:
dont do that

Reply:
sorry again , I used https://github.com/modeso-open-source/1lims-automation/blob/LIMS5300-4426/ui_testing/pages/testplan_page.py#L236
def update_upper_lower_limits_of_testunit(self, old_upper, old_lower):
I used it and it worked well, thank you for your note